notebooks/packages.clj

```diff
@@ -0,0 +1,47 @@
+^{:clay {:kindly/options {:kinds-that-hide-code #{:kind/md :kind/hiccup :kind/html :kind/tex}}}}
```

Comment:
Cannot we set these as the defaults in clay.edn? If not then we should propose to Daniel to make it possible...

Reply:
I think in theory we can: https://clojurians.zulipchat.com/#narrow/stream/151924-data-science/topic/Kindly.3A.20How.20to.20hide.20outputs/near/449986638

Reply:
added it to clay.edn and commented out the namespace declaration until it can be tested